src/phases/query/GetHostNameArray.yml

```diff
@@ -0,0 +1,24 @@
+SchemaVersion: 2018-07-01
```

Comment:
Why in a separate file? It's of no use outside of tsbs_query workloads and depends heavily on the schema of the collection used there (e.g. having exactly 100 hosts)